Debug App/Primer.io Debug App.xcodeproj/project.pbxproj

```diff
@@ -971,7 +977,6 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "";
 				PRODUCT_NAME = "Debug App";
 				PROVISIONING_PROFILE_SPECIFIER = "";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match Development com.primerapi.PrimerSDKExample";
```

Comment:
was removing the provisioning profile + code signing config intentional?

Reply:
Yes, we had problems with the latest release because I enabled Automatic and pushed it to the repo. 

cc @NQuinn27